packages/nouns-webapp/src/wagmi.ts

```diff
@@ -0,0 +1,42 @@
+import { find, pipe } from 'remeda';
+import { createConfig, http } from 'wagmi';
+import { mainnet, sepolia } from 'wagmi/chains';
+import { coinbaseWallet, injected, walletConnect } from 'wagmi/connectors';
+
+import { CHAIN_ID, WALLET_CONNECT_V2_PROJECT_ID } from './config';
+
+const activeChainId = Number(CHAIN_ID);
+
+const activeChain =
+  pipe(
+    [mainnet, sepolia],
+    find(chain => chain.id === activeChainId),
+  ) ?? sepolia;
+
+const mainnetRpcUrl = process.env.VITE_MAINNET_JSONRPC ?? '';
+const sepoliaRpcUrl = process.env.VITE_SEPOLIA_JSONRPC ?? '';
+
+export const config = createConfig({
+  chains: [activeChain],
```

Comment:
this is not working as intended.
When I'm connected to Sepolia on my wallet, wagmi is making contract calls to the Sepolia contract address on mainnet even when I have`VITE_CHAIN_ID=1` in my env, so the auction never loads

Reply:
> this is not working as intended. When I'm connected to Sepolia on my wallet, wagmi is making contract calls to the Sepolia contract address on mainnet even when I have`VITE_CHAIN_ID=1` in my env, so the auction never loads. If I switch to mainnet it works as intended

Looks like there was a hard-coded chain id; I updated it. I’ve been testing auctions and proposals on Sepolia today and everything works (aside from the loading bid button, which I’ll sort). During migration I tried to mimic ethers behavior wherever possible so the UI and other site functionality wouldn’t break. We definitely need a refactor to simplify the code moving forward, but this branch has grown way too big for that 😅